app/main.py

```diff
@@ -1,13 +1,40 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {"line": error.pop("line_number"), "column": error.pop("column_number"), "name": error.pop("code"),
```

Comment:
After the opening bracket, write all arguments on a new line, including the first one; also write the closing bracket on a new line